pkg/lib/config/config.go

```diff
@@ -293,8 +293,10 @@ func (c *AppConfig) validateLockout(ctx *validation.Context) {
 	minDuration, isMinDurationValid := c.Authentication.Lockout.MinimumDuration.MaybeDuration()
 	maxDuration, isMaxDurationValid := c.Authentication.Lockout.MaximumDuration.MaybeDuration()
 	if isMaxDurationValid && isMinDurationValid && minDuration.Seconds() > maxDuration.Seconds() {
-		ctx.Child("authentication", "lockout", "maximum_duration").
-			EmitErrorMessage("maximum_duration must be greater than or equal to minimum_duration")
+		ctx.Child("authentication", "lockout", "minimum_duration").EmitError("maximum", map[string]interface{}{
+			"maximum": c.Authentication.Lockout.MaximumDuration,
```

Comment:
Since `maximum` is a keyword for JSON number, the information it contains must also be number. Call `.Seconds()`.

Reply:
It seems a bit weird that I pass in a string "1m" and it tells me my input is `60` 🤔

Reply:
It does not matter in this case, because the error message we are going to show is something like `Lockout Duration must be less than or equal to Maximum Lockout Duration`. 

The change is just for conforming the protocol of `maximum`.

Reply:
Yes I understand. However it is still an error returned from the api, which the error message doesn't match the input.
Nvm, let me change it in this way first.